storage/cache.hpp

```diff
@@ -121,6 +128,24 @@ class CacheOptParser : public OptionsParser
          OptionFlags::RequiredArgument);
       register_option("S", "split", "", "Split biflows into uniflows",
          [this](const char *arg){ m_split_biflow = true; return true;}, OptionFlags::NoArgument);
+      register_option("fd", "frag-disabled", "", "Disable fragmentation cache",
+         [this](const char *arg){ m_disable_fragmentation_cache = true; return true;}, OptionFlags::NoArgument);
+      register_option("fs", "frag-size", "size", "Size of fragmentation cache, must be at least 1", [this](const char *arg) {
```

Comment:
Due to practical reasons, it is better to put default value into help string. This will help user to find it out.